src/Microsoft.Graph.Cli.Core/IO/JsonOutputFormatter.cs

```diff
@@ -1,20 +1,53 @@
 using System.CommandLine;
+using System.Text;
+using System.Text.Json;
 using Microsoft.Kiota.Cli.Commons.IO;
 using Spectre.Console;
 
 namespace Microsoft.Graph.Cli.Core.IO;
 
 public class JsonOutputFormatter : IOutputFormatter
```

Comment:
shouldn't bot those formatters live in the common lib so they could be reused by other CLIs?